### ISSUE TITLE
Wanted: Maintainer

### ISSUE THREAD

**Issue by jacopotarantino** [jacopotarantino/angular-match-media#18]
If anyone is interested in contributing heavily to the development of this plugin or would like to give back to the community in a meaningful way, I would love help maintaining this plugin. Comment here or send me an email if interested.

**Comment by zombiehugs** [comment]
Are you still looking for some one to maintain?

**Comment by jacopotarantino** [comment]
@zombiehugs absolutely. Any help at all is GREATLY appreciated and if you want to do some major work on this project or others I've got a long todo list haha. I also haven't been great about keeping up on issues/PR-vetting so contributions in that manner are probably the best for the project.

**Comment by zacronos** [comment]
@jacopotarantino I can't promise to dedicate a lot of time to this, but if you want to make me a maintainer I will at least try to keep things moving by soliciting, approving, and merging PRs.

**Comment by jacopotarantino** [comment]
@zacronos thanks so much for the offer! i've just sent you a collaborator email. any single minute that you can put into maintaining this project is deeply appreciated.

**Comment by zacronos** [comment]
Alright, I've accepted the github invite -- note that this will let me handle things on the github side, but if you want me to be able to cut corresponding npm releases, I'll need to be an npm collaborator as well.  Thanks!

**Comment by zacronos** [comment]
@jacopotarantino Since this is published on Bower as well as npm, I'll need permissions on both of those if you want me to cut new releases.

**Comment by jacopotarantino** [comment]
@zacronos what's your username on npm? i'll add you. i think bower is done for now so we should probably remove support for that if anything.
